native/libcst/src/parser/grammar.rs

```diff
@@ -498,6 +499,13 @@ parser! {
                     make_try(kw, b, ex, el, f)
             }
 
+        // Note: this is separate because TryStar is a different type in LibCST
+        rule try_star_stmt() -> TryStar<'a>
```

Comment:
The official grammar has this as a variant under `try_stmt`. We're sure it's worth diverging? I can see some value in it since the actual AST node variant is different, but there's a price we'll pay keeping things in sync for every rule that diverges.

Reply:
Not 100% sure it's worth it, I was a bit lazy and wanted to avoid an additional enum layer just so we can have a single rule.

Reply:
Oh are the rules tied more tightly to types in the rust PEG library than in C?

In CPython it's a single PEG variant, but two different AST types still